### DISCUSSION TITLE
Distribution Roadmap

### DISCUSSION THREAD

**Discussion by bjdev86** [AndronixApp/AndronixOrigin#245] [Q&A]
Hello, question: What is the roadmap for distro-upgrades? 

For example, the stable distribution for Debian that gets installed is version 10 (Buster), but the current stable version (I believe it's stable) is version 12 (Bookworm). 

I'm new to the "running-Linux-on-an-ARM-processor" world, and from the little research I have done I'm seeing that it may be that we're limited to distro upgrades that are released for ARM processors?

What is Andronix's roadmap/plan to keep users up to date with new distro releases? OR What can users do to try an keep themselves up to date with new distro releases? Is it simply using the Modd OS versions that are available for purchase?

Thanks for all your effort in working on this project. It's really amazing!

**Comment by elvisisvan** [comment]
what's `distro-upgrades`?
